src/index.js

```diff
@@ -56,10 +56,41 @@ newTask.addEventListener('click', (e) => {
     const k = parseInt(e.target.id, 10);
     const newArray = tasks.filter((Objects) => Objects.index !== k);
 
-    for (let i = k - 1; i < newArray.length; i += 1) {
-      newArray[i].index -= 1;
+    for (let i = 0; i < newArray.length; i += 1) {
+      newArray[i].index = 1;
+    }
+    for (let i = 0; i < newArray.length; i += 1) {
+      newArray[i].index += i;
     }
     localStorage.setItem('list', JSON.stringify(newArray));
+  } else if (e.target.checked) {
+    const k = parseInt(e.target.id, 10);
+    tasks[k - 1].completed = true;
+    console.log(e.target.checked.id);
+    localStorage.setItem('list', JSON.stringify(tasks));
```

Comment:
- [ ] Please remove all `console.log()` from your codebase. Having it in your committed code makes your work look untidy 🙏